fix: This is my third attempt at doing this entry. (#2720)
Codecademy/docs: content/markdown/concepts/highlights/highlight.md

Reviewer: Path needs to be changed to: `content/markdown/concepts/highlight/highlight.md`  (remove the `s`)

---
Title: 'highlight'

> Description: 'Highlights are when two equal signs are placed in between a piece(s) of text, resulting a yellow background encompassing said text.'

Reviewer: ```suggestion
Description: 'Highlights are when two pairs of equal signs enclose a piece of text, resulting a yellow background encompassing said text.'
```

> Subjects:
>   - 'Web Development'
>   - 'Developer Tools'
> 
> Tags:
>   - 'Documentation'
> 
> CatalogContent:
>   - 'learn-html'
>   - 'path/front-end-engineer-career-path'
> ---

Reviewer: ```suggestion
Subjects:
  - 'Web Development'
  - 'Developer Tools'
Tags:
  - 'Documentation'
CatalogContent:
  - 'learn-html'
  - 'path/front-end-engineer-career-path'
---
```
Remove blank lines from metadata

> In Markdown, **highlight** is where parts of a text appears in a yellow background after being inserted between two equal signs (==).

Reviewer: ```suggestion
In Markdown, **highlight** is where parts of a text appears in a yellow background after being inserted between two pairs of equal signs (`==`).
```

> **Note:** This feature is not offered in all Markdown processors so consider reading through the instruction guide before completing.

Reviewer: ```suggestion
> **Note:** This feature is not offered in all Markdown processors so consider reading through the instruction guide before completing.
```
